#!/usr/bin/python
PACKAGE = "ucb_jaco_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("p_gain", double_t, 0, "Proportional gain.", 5, 0, 1000)
gen.add("i_gain", double_t, 0, "Integral gain.",     0, 0, 1000)
gen.add("d_gain", double_t, 0, "Derivative gain.",   0, 0, 1000)

exit(gen.generate(PACKAGE, "ucb_jaco_control", "PIDGains"))